### ISSUE TITLE
Scroll, Layout Vertical Paddings and Stack Overflow

### ISSUE THREAD

**Issue by r3d9u11** [RealyUniqueName/StablexUI#234]
Hi. I have same error after upgrading openfl and lime
This error appears when I trying to define seleral properties "layout:Row-paddingTop" & "layout:Row-paddingBottom"

Error log: [err.pdf](https://github.com/RealyUniqueName/StablexUI/files/454720/err.pdf) (40856 lines o_O)

Haxe: 3.2.1
Neko: 2.1.0
Haxelib list: http://pastebin.com/15zVQRCU
OS: Linux Mint 18 x64

Main.hx:

```
package;

import openfl.display.*;

import ru.stablex.ui.UIBuilder;
import ru.stablex.ui.widgets.Floating;

class Main extends Sprite {
	public function new () {
		super();
		
		this.stage.addChild(this);
		
		UIBuilder.init();
		var window:Floating = UIBuilder.buildFn("Main.xml")();
		window.show();
	}
}
```

Main.xml
```
<?xml version="1.0" encoding="UTF-8"?>
<Floating widthPt="100" heightPt="100">
	<Scroll
		widthPt="100"
		heightPt="100"
		skin:Paint-color="0xC0C0C0"
		vBar-slider-skin:Paint-color="0x002200"
		hBar-slider-skin:Paint-color="0x002200"
		hBar-skin:Paint-color="0x777777"
		vBar-skin:Paint-color="0x777777"
		vBar-visible="true"
		hBar-visible="true">
		<VBox
			align="'top,left'"
			layout:Row-rows="[-1]"
			layout:Row-paddingLeft="10"
			layout:Row-paddingTop="10"
			layout:Row-paddingRight="20"
			layout:Row-paddingBottom="20">
			<Widget
				skin:Paint-color="0xFFFFFF"
				skin:Paint-border="3"
				skin:Paint-borderColor="0x000000" />
		</VBox>
	</Scroll>
</Floating>
```

if try to delete the lines
```
layout:Row-paddingTop="10"
layout:Row-paddingBottom="20"
```
then this UI works fine

_In the older lime and openfl this UI works, but vertical scrollbar don't shows at startup when size of Widget more, than current window (unlike of horizontal scroll, it works fine). Vertical scrollbar will be visible only after first window resizing.

I think that abnormal behavior of vertical scroll and child-padding with stack overflow error somehow connected._

Thanks.

**Comment by RealyUniqueName** [comment]
Fixed in c48d78c

This happened because `VBox` by default automatically recalculates height if child widget resized. And `Row` automatically resizes child widget on `VBox` resize.
It's strange it worked with previous OpenFL. Perhaps in old versions OpenFL blocked dispatching the same event multiple times per frame.

**Comment by r3d9u11** [comment]
After updating from git it works fine now. Thank you!
